src/diffusers/models/prior_transformer.py

```diff
@@ -34,14 +34,31 @@ class PriorTransformer(ModelMixin, ConfigMixin):
         num_attention_heads (`int`, *optional*, defaults to 32): The number of heads to use for multi-head attention.
         attention_head_dim (`int`, *optional*, defaults to 64): The number of channels in each head.
         num_layers (`int`, *optional*, defaults to 20): The number of layers of Transformer blocks to use.
-        embedding_dim (`int`, *optional*, defaults to 768):
-            The dimension of the CLIP embeddings. Image embeddings and text embeddings are both the same dimension.
-        num_embeddings (`int`, *optional*, defaults to 77): The max number of CLIP embeddings allowed (the
-            length of the prompt after it has been tokenized).
+        embedding_dim (`int`, *optional*, defaults to 768): The dimension of the model input `hidden_states`
+        num_embeddings (`int`, *optional*, defaults to 77):
+            the number of embeddings of the model input `hidden_states`
```

Comment:
Are these not CLIP embeddings?

Reply:
Let's try to keep it general here though and not call it CLIP, think the prior transformer can very well also work with other models than CLIP (e.g. CLAP)